TicketsDemo/App_Data/carriages.csv

```diff
@@ -0,0 +1,8 @@
+Id,Type,DefaultPrice,TrainId,Number
```

Comment:
please add resource files to .gitignore (but previosly - delete them from filesystem)